CrisisTextLine/Elmer
# New new drop melted users

What's this PR do?

Fixes a bug where Typeform submissions weren't being graded; the Typeform submissions weren't showing up as being in a given course. John changed updateCanvas code, and Assaf’s Typeform code was using updateCanvas code.

Where should the reviewer start?

updateCanvas.js, specifically canvas.retrieveEnrollment and canvas.scrapeCanvasEnrollment.

How should this be manually tested?

Assaf and I tested by:
1. Running node www/app.js.
2. Exposing our local port (3000) using ngrok.
3. Changing line 70 of www/canvas/index.js to read:
email: 'john@crisistextline.org' instead of
email: 'heather@crisistextline.org'
4. Creating a fake Typeform assignment in a course in Canvas as an admin.
5. Submitting that assignment as a user.
6. Checking ngrok, www/app.js, and the email address to make sure everything went through okay.

Any background context you want to provide?

What are the relevant tickets?

[INT-401
](https://admin.crisistextline.org/jira/browse/INT-401)
Questions:
